src/drivers/intel/ssp/ssp.c

```diff
@@ -67,23 +67,44 @@ static void ssp_empty_tx_fifo(struct dai *dai)
 		ssp_write(dai, SSSR, sssr);
 }
 
+/*
+ * Empty SSP receive FIFO before DMA startup and
+ * ensure the DMA request is not already raised when
+ * DMA is started. If TX is not active, this is a no-op.
```

Comment:
I'm confused a bit: isn't receive FIFO usually the FIFO where data arrives from the interface, i.e. for capture? Why is it talking about TX then?